### ISSUE TITLE
Hyperparamers for finetuning

### ISSUE THREAD

**Issue by zwzhang123** [Mandar-Sharma/Project2021#1]
Thanks for the interesting work.

Can you provide more details for finetuning?

**Comment by Mandar-Sharma** [comment]
Thanks for taking interest in our work. Besides the values for p and k for the top-p/top-k sampling and the training epochs (outlined in the paper), the model finetuning hyperparamterers (like the learning rate, scheduling) are huggingface defaults.
